### DISCUSSION TITLE
It is possible to use that allows access to read/write (i.e. copy/paste) access to the system clipboard. Using buttons?

### DISCUSSION THREAD

**Discussion by Haykkonen** [discordjs/discord.js#7657] [Q&A] [answered]
Hello, I would like to know if it is possible to use the read/write function in discord using buttons like the example below: when clicking on the button, the player copies a string provided by the bot

![image](https://user-images.githubusercontent.com/60020692/158088684-f60d9109-3d7b-4245-a369-ae447542d76e.png)

**Comment by KhafraDev** [comment, marked as answer]
No. There are suggestions to add this behavior (https://github.com/discord/discord-api-docs/discussions/3374, https://github.com/discord/discord-api-docs/discussions/3634), but it is "unlikely" to ever happen.
